### PR TITLE
Add option to remove labels, remove outline on drag

### PR DESCRIPTION
**Pull request by fellmirr**
Adds an opiton to remove labels from the slider. Removes ugly otline on drag in chrome on Windows 10. Also adds basic typescript prop typings.

I merged with another fork to include some of their bugixes, but ended up getting a lot of changes they made, which was unintended. I tried to revert the merge, alas with no success 🤷 Meaning, all the comits not made by me are *not* actually included, but still appear in the commit log.

### CODE REVIEW

**Comment by samhogg** [comment]
Cool, thanks! I have been neglecting this project big time but I'll take some time tomorrow to review and push this onto npm.

File: src/components/Slider.js [hunk 1]
```diff
@@ -154,6 +155,7 @@ class Slider extends Component {
               onKeyDown={this.keyDown.bind(this, i)}
               transform={`translate(${this.props.scale(m)}, 0)`}
               key={`handle-${i}`}
+              style={{outline: 'none'}}
```
Comment (samhogg) [inline src/components/Slider.js]: There should probably add some form of focus styling for accessibility

**Comment by fellmirr** [comment]
That's awesome!
It would be super usefull if you publish an updated version, as I'm strugeling to get google cloud build to work correctly with submodules, hehe...

🔨⚙️

**Comment by samhogg** [comment]
@fellmirr sorry for the delay, published `v2.0.15`

**Pull request merged**